src/ray/rpc/gcs_server/gcs_rpc_client.h

```diff
@@ -258,8 +268,8 @@ class GcsRpcClient {
                              heartbeat_info_grpc_client_, /*method_timeout_ms*/ -1, )
 
   /// Check GCS is alive.
-  VOID_GCS_RPC_CLIENT_METHOD(HeartbeatInfoGcsService, CheckAlive,
-                             heartbeat_info_grpc_client_, /*method_timeout_ms*/ -1, )
+  VOID_RPC_CLIENT_METHOD(HeartbeatInfoGcsService, CheckAlive, heartbeat_info_grpc_client_,
+                         /*method_timeout_ms*/ 100, )
```

Comment:
Why do we change the timeout?

Reply:
When gcs_server dead or restart, we expect the `CheckAlive` call fails or returns quickly, since we try again and again in a loop. Don't worry too much about the additional impact of timeout 100ms, since only be used in GcsClient reconnection at present.

Reply:
Can we not include it in this pr and update it separately? After running some tests. This seems to be unrelated changes?